Firefox Selection Fix.sh

```diff
@@ -6,8 +6,23 @@
 return_wd="$PWD"
 reason_already_root='already_root'
 description='The Firefox Selection Fix script disables the broken clickSelectsAll behavior of Firefox. Make sure Firefox is up-to-date and closed'
-firefox_dir=$(whereis firefox | cut -d ' ' -f 2)
-fallback_firefox_dir='/usr/lib/firefox' # Fallback path: put your Firefox install path here. The install path includes the `firefox` binary and a `browser` directory.
+
+### Find out the operating system
```

Comment:
You’re introducing a lot of comments in the code, most of which aren’t that useful, _especially_ this one. Consider explaining things in the README if absolutely necessary.